src/index.ts

```diff
@@ -392,6 +402,38 @@ async function removeDockerPodmanImageStroage(): Promise<void> {
     }
 }
 
+async function checkIfManifestsExists(): Promise<boolean> {
```

Comment:
If at least one manifest is found, log somewhere the list of images that are manifests